README.md

```diff
@@ -721,23 +694,24 @@ The complete list of endpoints, with a short description and the signature speci
 
  ***NOTE*** Most of the endpoints update the config file given in the Dockerfile. If you don't have this file mounted (see section [Persisting changes](#persisting-changes)), these changes will be inside your container and will be lost after stopping it.
 
+
 ### Interacting with the processors' generated information
 
 #### Generated information
 The generated information can be split into 3 categories:
-  - `Raw data`: This is the most basic level of information. It only includes the results of the detector, classifier, tracker, and any configured post-processor step.
-  - `Metrics data`: **Only written if you have enabled the metrics periodic task** (see [section](#change-the-default-configuration)). These include metrics related to occupancy, social-distancing, and facemask usage; aggregated by hour and day.
-  - `Notifications`: Situations that require an immediate response (such as surpassing the maximum occupancy threshold for an area) and need to be notified ASAP. The currently supported notification channels are email and slack.
+  - `Raw data`: This is the most basic level of information. It only includes the results of the detector, classifier, tracker, and any configured post-processor step. All the video information generated (such as the live feed) is **stored inside the edge device and is never sent outside it**. 
```

Comment:
```suggestion
  - `Raw data`: This is the most basic level of information. It only includes the results of the detector, classifier, tracker, and any configured post-processor step. All the video information generated (such as the live feed) is **stored inside the edge device and is never sent to cloud or any other storage**. 
```
It might be sent to the browser of the users